src/storage/fileset.proto

```diff
@@ -21,6 +21,14 @@ message DeleteFile {
   string path = 1;
 }
 
+// TODO: Src and dst naming?
+// TODO: Append?
+message CopyFile {
+  string fileset_id = 1;
```

Comment:
It would be good to put some validation rules on each field, just to make obvious errors obvious to the caller.  See examples in pfs.proto; here `[(validate.rules).string.len = <fileset id len>]` would be helpful.  src and dst should probably have `min_len = 1`.  Available validation rules are in `src/protoextensions/validate.proto#StringRules`.

Reply:
I will do this in a followup PR for the whole fileset API.